### ISSUE TITLE
bulletgraph-configuring graph background issues

### ISSUE THREAD

**Issue by IGvaleries** [Infragistics/convertwire#208]
file: bulletgraph-configuring-the-background.adoc

1) space between control name and TM:
![converted](https://cloud.githubusercontent.com/assets/18487709/14648963/ede9b304-0632-11e6-8736-86e5cf50f65c.PNG)

2) [_Ref357787115] should be removed after Topics:

![converted](https://cloud.githubusercontent.com/assets/18487709/14648896/9583e158-0632-11e6-8f03-119a72cb739f.PNG)

**Comment by IGvaleries** [comment]
a similar issue with ref number also appears in bulletgraph-configuring-the-scale.adoc:
![converted](https://cloud.githubusercontent.com/assets/18487709/14649795/9527f9ac-0636-11e6-974d-7d8f88c2abe4.PNG)

**Comment by ig-ealbert** [comment]
For both "Configuring the Background" and "Configuring the Scale", this is an extra internal anchor in the HTML (from Document X) and should be fixed in the topic itself.  Submit a TFS bug and I can fix it in English.

```
<ul>
    <li><a href="#_Ref362875752">Topics</a><a href="#_Ref357787115"></a></li>
</ul>
```

**Comment by craigshoemaker** [comment]
I have labeled this to fix in DocX. Thanks, @ig-ealbert!

**Comment by ig-ealbert** [comment]
The second part is already fixed in #210 and TFS bug 219301.
Created TFS 219324 for the first part with the extra internal anchor.
The TM space I think is intentional as otherwise it was getting italicized and this was the "fix".  @craigshoemaker can confirm that.

**Comment by ig-ealbert** [comment]
Fixed in TFS.
